raresGabriel01/home-assignment
# [ISSUE-15] Unit tests for t-grid component

# Issue link
#15 
# Description
Implemented unit tests for the `t-grid` component using `@angular/testing/core`.
Tried to provide an exhaustive list of use cases.
# How to test?

1. open a terminal
2. `cd` into the root of the project
3. run `ng test`
4. ensure all tests pass

# Attachments
